db/migrate/20190228160047_add_value_type_and_resource_type_to_template_input.rb

```diff
@@ -0,0 +1,6 @@
+class AddValueTypeAndResourceTypeToTemplateInput < ActiveRecord::Migration[5.2]
+  def change
+    add_column :template_inputs, :value_type, :string, :default => 'plain'
```

Comment:
`:null => false` would be also good now, since all will have at least plain, in case someone in future will try to define template_input without this, it will yell :-)